protoc-gen-swagger/genswagger/template.go

```diff
@@ -10,10 +10,14 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/golang/protobuf/proto"
 	pbdescriptor "github.com/golang/protobuf/protoc-gen-go/descriptor"
 	"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor"
+	swagger_options "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/options"
 )
 
+var swaggerExtrasRegexp = regexp.MustCompile(`(?s)^(.*[^\s])[\s]*<!-- swagger extras start(.*)swagger extras end -->[\s]*(.*)$`)
```

Comment:
swaggerExtrasRegexp... what's that? Could you add  a comment here explaining it's purpose? It doesn't seem to get used anywhere else